examples/colab/tf2_hub_single_image_super_resolution_ESRGAN.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>Do you have to install scipy?</p><p> </p><p>It is preinstalled in the colab.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Scipy Version 1.2.x supports imresize().</p><p>However I found an alternative to Scipy, So I'll replace it with this new function</p><p> </p>